src/test/java/io/quarkus/search/app/SearchServiceTest.java

```diff
@@ -42,18 +50,18 @@ protected int managementPort() {
         }
     }
 
-    private SearchResult<SearchHit> search(String term) {
+    private SearchResult<GuideSearchHit> search(String term) {
         return given()
                 .queryParam("q", term)
-                .when().get()
+                .when().get(GUIDES_SEARCH)
                 .then()
                 .statusCode(200)
                 .extract().body().as(SEARCH_RESULT_SEARCH_HITS);
     }
 
     @BeforeAll
     void waitForIndexing() {
-        Awaitility.await().untilAsserted(() -> {
+        Awaitility.await().timeout(Duration.ofMinutes(5)).untilAsserted(() -> {
```

Comment:
It shouldn't be that slow... We just unzip a git repository and create a local clone...

On the other hand the default is 15 seconds, which does seem a bit tight.

I'll add a lower timeout to https://github.com/quarkusio/search.quarkus.io/pull/55.